src/backend/db/users.js

```diff
@@ -25,4 +25,13 @@ export const users = [
     createdAt: formatDate(),
     updatedAt: formatDate(),
   },
+  {
+    _id: uuid(),
+    firstName: "test",
+    lastName: "test",
```

Comment:
Let the email and password be whatever, give proper first name. It does not look good when you display `Test` as the username

Reply:
changed this thanks!